src/main/java/ch/njol/skript/expressions/ExprTimePlayed.java

```diff
@@ -101,5 +104,14 @@ public Class<? extends Timespan> getReturnType() {
 	protected String getPropertyName() {
 		return "time played";
 	}
+
+	private Timespan getTimePlayed(OfflinePlayer offlinePlayer) {
+		if (IS_OFFLINE_SUPPORTED) {
+			return Timespan.fromTicks_i(offlinePlayer.getStatistic(Statistic.PLAY_ONE_MINUTE));
+		} else if (offlinePlayer.isOnline()) {
+			return Timespan.fromTicks_i(offlinePlayer.getPlayer().getStatistic(Statistic.PLAY_ONE_MINUTE));
+		}
+		return new Timespan(0);
```

Comment:
I don't think the expression should return 0 ticks when the player is offline and offline players aren't supported, I think it should return null. The player being offline doesn't mean they've never played, it just means we don't know it.